content/en/_index.md

```diff
@@ -19,7 +19,8 @@ weight: 10
     <p>
       Guides and resources to help you get a service into production and the hands of customers.
```

Comment:
@garrettthomaskth this text should be ` Guides and resources to help you get a service into production and into the hands of customers.`

Reply:
:+1: